test/helpers/helpers.js

```diff
@@ -37,6 +38,16 @@ beforeEach(() => {
           return result;
         }
       };
-    }
+    },
+    toRenderExact: () => ({
+      compare: async(spec, expected) => {
+        const result = {};
+        const actual = marked(spec.markdown, spec.options);
+
+        result.pass = assert.deepStrictEqual(expected, actual) === undefined;
```

Comment:
```suggestion
        result.pass = assert.strictEqual(expected, actual) === undefined;
```

since `expected` and `actual` are always strings there is no need to do a deep equal.

Reply:
done